akka-projection-grpc/src/main/scala/akka/projection/grpc/internal/ProtobufProtocolConversions.scala

```diff
@@ -0,0 +1,244 @@
+/*
+ * Copyright (C) 2009-2023 Lightbend Inc. <https://www.lightbend.com>
```

Comment:
2023, to be picky

Reply:
I wrongfully cast blame on the plugin, now correct in [5571754](https://github.com/akka/akka-projection/pull/932/commits/55717547caf9e83ff40b3a7cbe78ecc62175f7fe)